Features/LootItems.cs

```diff
@@ -35,9 +36,14 @@ internal class LootItems : PointOfInterests
 		[ConfigurationProperty]
 		public bool ShowPrices { get; set; } = true;
 
+		[ConfigurationProperty]
+		public bool TrackWishlist { get; set; } = true;
+
 		public override float CacheTimeInSec { get; set; } = 3f;
 		public override Color GroupingColor => Color;
 
+		public GClass2903? Wishlist { get; set; } = null;
```

Comment:
This will break the compilation next update. We should never reference a GClassXXXX type, they are auto numbered by the deobfuscator.

Reply:
That makes sense. What would be a good method to define that Wishlist variable?

Reply:
Given we are in `RefreshData`, it's ok to retrieve the instance every call. Let me write that down.

Reply:
Got it. I tried also doing something like this but my spidey sense says that's not a good. Should I move it back down to both the TryAddRecordIfTracked and place it also in the Commands.cs?

```cs
public object? Wishlist { get; set; } = null;```

Reply:
Good copy. Let me take a look.